sensor_msgs/Image face
---
string face_emotion
